00-repos-centos7.ks

```diff
@@ -1,5 +1,6 @@
 repo --name=centos --mirrorlist=http://mirrorlist.centos.org/?release=7&arch=$basearch&repo=os
 repo --name=centos-updates --mirrorlist=http://mirrorlist.centos.org/?release=7&arch=$basearch&repo=updates
+repo --name=centos-sclo --mirrorlist=http://mirrorlist.centos.org/?release=7&arch=x86_64&repo=sclo-rh
```

Comment:
Why this is needed? I mean, smart proxy will not be part of this repo. Where the kickstart is supposed to find it? Foreman-el7 repo?

Reply:
this repo is a dependency of `foreman-proxy` now

Reply:
Nit: to avoid confusion I'd name it `sclo-rh` since there's also a repo `sclo` for CentOS.
```suggestion
repo --name=centos-sclo-rh --mirrorlist=http://mirrorlist.centos.org/?release=7&arch=x86_64&repo=sclo-rh
```